lib/enumerate_it/base.rb

```diff
@@ -1,6 +1,7 @@
 require 'forwardable'
 
 module EnumerateIt
+  # rubocop:disable Metrics/ClassLength
```

Comment:
@lucascaton I put this exception because it doesn't make sense to refactor this class.

Reply:
I'll think about a solution later, no worries 🙂